Disable pam_userdb module.

saukko opened the pull request:
[pam] Disable pam_userdb module. Fixes JB#57452

We do not use the module, and it also depends on Berkley DB and this way
we can build pam without such build dep.

Signed-off-by: Marko Saukko <marko.saukko@jolla.com>

rainemak reviewed (approved):
Didn't test building of pam and running it but change itself looks good to my eye. Special thanks for cleaning up Groups.

saukko committed:
Disable pam_userdb module.

[pam] Disable pam_userdb module. Fixes JB#57452

We do not use the module, and it also depends on Berkley DB and this way
we can build pam without such build dep.

Signed-off-by: Marko Saukko <marko.saukko@jolla.com>
